metecho/api/serializers.py

```diff
@@ -548,6 +550,7 @@ def create(self, validated_data):
             validated_data["assigned_dev"] = user.github_id
 
         task = super().create(validated_data)
+        task.notify_created(originating_user_id=str(user.id))
```

Comment:
@jerivas I decided to trigger this via the serializer instead of the model `save` handler, but let me know if you prefer otherwise.